locale/es_CL/LC_MESSAGES/client.po

```diff
@@ -1961,7 +1961,7 @@ msgstr "No debe ser tu email"
 
 #: .es5/templates/partial/password-strength-balloon.mustache:10
 msgid "Must not match this <a %(escapedCommonPasswordLinkAttrs)s>list of common passwords</a>"
-msgstr "No debe coincidir con la <a %(escapedcommonpasswordlinkattrs)s=\"\">lista de contraseñas comunes</a>"
+msgstr "No debe coincidir con la <a %(escapedCommonPasswordLinkAttrs)s=\"\">lista de contraseñas comunes</a>"
```

Comment:
@shane-tomlinson what's going on with `s=\"\"` after the `escapedCommonPasswordLinkAttrs` ? 😕